app/serializers/api/order_detailed_serializer.rb

```diff
@@ -5,8 +5,16 @@ class OrderDetailedSerializer < Api::Admin::OrderSerializer
     has_one :bill_address, serializer: Api::AddressSerializer
 
     has_many :line_items, serializer: Api::LineItemSerializer
-    has_many :adjustments, serializer: Api::AdjustmentSerializer
 
     has_many :payments, serializer: Api::PaymentSerializer
+
+    attributes :adjustments
+
+    def adjustments
+      adjustments = object.all_adjustments.where(
+        "adjustable_type IN ('Spree::Order','Spree::Shipment')"
```

Comment:
why not `where(adjustable_type: %w(Spree::Order Spree::Shipment))`. I always feel that getting to SQL literal is giving up on OOP and all its benefits. That's why I see it as a last resort only.

Reply:
Done :+1: